### PR TITLE
docs: add 8.0.0-beta1 release notes

### PR DESCRIPTION
**Pull request by bmorelli25**
This PR adds the `8.0.0-beta1` release notes.

`8.0.0-alpha1` notes were easy because it was the first version in the `8.0.0*` release. `8.0.0-alpha2` notes were easy because there were no changes. But I got a bit stuck on these. Is there an easy way to see which commits will be in these pre-releases before the release?

### CODE REVIEW

File: changelogs/8.0.asciidoc [hunk 1]
```diff
@@ -3,9 +3,24 @@
...
 
+[float]
+[[release-notes-8.0.0-beta1]]
+=== APM version 8.0.0-beta1
+
+https://github.com/elastic/apm-server/compare/v8.0.0-alpha2\...v8.0.0-beta1[View commits]
+
+[float]
+==== Added
+- Add support for CloudWatch metric streams through firehose endpoint {pull}6380[6380]
```
Comment (simitt) [inline changelogs/8.0.asciidoc:18]: Can we add an experimental batch here please?

Comment (bmorelli25) [reply changelogs/8.0.asciidoc:18]: Yup!

Comment (bmorelli25) [reply changelogs/8.0.asciidoc:18]:
```suggestion
- experimental:[] Add support for CloudWatch metric streams through firehose endpoint {pull}6380[6380]
```

**Comment by bmorelli25** [comment]
Agh, sorry! I tagged the wrong team for review.

**Review by marclop — APPROVED** [review]

**Pull request merged**